frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1589,15 +1600,18 @@ impl<T: Config> Pallet<T> {
 		<QueuedSolution<T>>::take()
 			.ok_or(ElectionError::<T>::NothingQueued)
 			.or_else(|_| {
-				T::Fallback::instant_elect(None, None)
-					.map_err(|fe| ElectionError::Fallback(fe))
-					.and_then(|supports| {
-						Ok(ReadySolution {
-							supports,
-							score: Default::default(),
-							compute: ElectionCompute::Fallback,
-						})
+				T::Fallback::instant_elect(
+					ElectionBounds::new_unbounded(),
```

Comment:
note that we don't overwrite a bound here but the fallback itself can have a bound. This is why we put unbounded here (`onchain::BoundedExecution`).